### PR TITLE
Option to install the drupal-vm

### PR DESCRIPTION
**Pull request by becw**
Notes so far...

1. setup time is > 10 minutes (13 mins for me this time, but it will depend on your connection)... but the first time it's more like 35 minutes because you have to download the base box
2. I think this installs and configures solr, but we might also have to start the solr service? not sure, but I can't access solr at http://projectname.drupalvm:8983/solr

To test:

* On a project, require this branch of `the-vagrant` (please don't do this permanently, as this branch will be deleted if the PR is merged) -- `composer require --dev palantirnet/the-vagrant:dev-drupal-vm`
* Destroy your existing box: `vagrant destroy`
* Install the drupal-vm: `vendor/bin/the-vagrant-installer-drupalvm`
  * This will prompt you for a project name, which defaults to the name of the project directory
* Observe two new/updated files: `Vagrantfile` and `conf/drupalvm/config.yml`
* Bring up your vagrant box: `vagrant up` (this will take 10-45 minutes)
* Log in to your vagrant: `vagrant ssh`
* Find your project root: `cd /var/www/drupalvm` and observe your project stuff there
* You will need to update your mysql db, username, and password in your `conf/build.default.properties` file to use `drupal`/`drupal`/`drupal`
* Try running drush and phing and stuff
* Visit your site in a browser: http://projectname.drupalvm

### CODE REVIEW

**Comment by liberatr** [comment]
drush doesn't work for me inside the vm as it's currently configured. I had to `cd` into `sites/default` in order to use drush.

**Comment by labbydev** [comment]
I'd recommend including updates to the `README.md` for this as well. Also I had to update the installer script so that I included `--ingnore-platform-reqs` when running composer. Since this won't be run in the VM that might be a change we want to make in the code. I also noticed I need to set (read increase) `config.vm.boot_timeout` so that might be something we want to include by default as well.

File: conf/drupalvm/config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  # - blackfire
+  - drupalconsole
+  - drush
+  # - elasticsearch
+  - java
+  - mailhog
+  - memcached
+  # - newrelic
+  # - nodejs
+  - pimpmylog
```
Comment (becw) [inline conf/drupalvm/config.yml:24]: This kills me. Why do we have to deal with crap names like this?

File: conf/drupalvm/config.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  # - redis
+  # - ruby
+  # - selenium
+  - solr
+  - varnish
+  # - xdebug
+  # - xhprof
+
+# Acquia Solr version
+solr_version: "4.5.1"
```
Comment (becw) [inline conf/drupalvm/config.yml:34]: @labbydev @liberatr Does Solr work on the boxes you've set up?

**Comment by becw** [comment]
@liberatr - I bet this isn't set up to add `vendor/bin` to the `$PATH`, nor to use the bash wrapper that `the-vagrant` uses for the devkit in order to set the drushrc file. Is drush installed globally on the drupalvm? If so, it might be a case of specifying the drushrc file.

@labbydev What README changes would you find helpful?

**Comment by labbydev** [comment]
The `README` should include instructions on how to use this

**Pull request closed without merging**